readme.md

```diff
@@ -0,0 +1,161 @@
+# Remark for Visual Studio Code
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+**[remark][]** is a tool that transforms markdown with plugins.
+These plugins can inspect and change your markup.
+You can use remark on the server, the client, CLIs, deno, etc.
+This plugin for Visual Studio Code lets you format and lint markdown files using
+remark.
+
+## Contents
+
+*   [When should I use this?](#when-should-i-use-this)
+*   [Use](#use)
+    *   [Formatting](#formatting)
+*   [Plugins](#plugins)
+*   [Syntax](#syntax)
+*   [Compatibility](#compatibility)
+*   [Security](#security)
+*   [Contribute](#contribute)
+*   [License](#license)
+
+## When should I use this?
+
+If your project is checked using [`remark-cli`][remark-cli] and you use Visual
+Studio Code to edit markdown files, it’s recommended to use this extension.
+
+## Use
+
+To start using this for a project, [`remark-cli`][remark-cli] needs to be
+configured first.
+For example, install `remark-cli` and `remark-preset-lint-recommended`:
+
+```sh
+npm install remark-cli remark-preset-lint-recommended
+```
+
+Now create a file named `.remarkrc.json` with the following content:
+
+```json
+{
+  "plugins": [["preset-lint-recommended"]]
+}
+```
+
+Now any syntax which doesn’t comply with `remark-preset-lint-recommended` will
+be reported in the *Problems* pane in your editor.
+
+### Formatting
+
+This extension can be used as a formatter for markdown files.
+
+To format a file, first press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>.
+Search for and select *“Format Document With…”*.
+Now select *“Remark”*.
+
+To make this the default formatter, add the following to your `settings.json`
+(Open with <kbd>Ctrl</kbd>+<kbd>,</kbd>):
+
+```json
+{
+  "[markdown]": {
+    "editor.defaultFormatter": "unifiedjs.vscode-remark",
+  }
+}
+```
+
+Optionally enable format on save:
+
+```json
+{
+  "[markdown]": {
+    "editor.defaultFormatter": "unifiedjs.vscode-remark",
+    "editor.formatOnSave": true
+  }
+}
+```
+
+## Plugins
+
+The **remark** ecosystem has a variety of plugins available.
+Most notably you’ll want to check out [`remark-lint`][remark-lint].
+A curated list of plugins can be found [here][list-of-plugins].
```

Comment:
“here” as link text is confusing. The “List of plugins” should be linked instead. Perhaps reword as: `See the [List of plugins][...] for all remark plugins`